src/test/java/capstone/mfslbackend/service/TeamServiceTest.java

```diff
@@ -165,13 +160,13 @@ public void testCreateTeamById_SuccessExistsAlready() throws IOException {
         teamResponse1.setName(teamName);
         teamResponse1.setLogo(teamName);
 
-        teamsResponses.add(new TeamsResponse(teamResponse1));
+        teamsResponses.add(new TeamsResponse(teamResponse1, null, null));
         teamsContainer.setResponse(teamsResponses);
 
         Mockito.when(apiService.getRequest(Mockito.any(), Mockito.eq(TeamsContainer.class)))
                 .thenReturn(teamsContainer);
         Long newTeamId = 2L;
-        Team team2 = new Team(newTeamId, teamName, teamName, new ArrayList<>());
+        Team team2 = new Team();
```

Comment:
There's no way the test passes like this? you got rid of the attributes being tested on lines 175,176,177